### DISCUSSION TITLE
Clipboard manager using cliphist?

### DISCUSSION THREAD

**Discussion by dkeithdj** [Aylur/ags#488] [Ideas]
Is it possible to have implementation using `cliphist`?

How does ags accept inputs?

such as `cliphist list | dmenu | cliphist decode | wl-copy`. And instead of `dmenu` it will be replaced with `ags -t clipboard`

**Comment by joncrangle** [comment]
I've been taking inspiration from @folke's dotfiles for my implementation to use the launcher as a cliphist manager, shell launcher and emoji picker.

This should hopefully help you get started: https://github.com/folke/dot/blob/master/config/ags/widget/launcher/Cliphist.ts
